src/tool/hpcrun/gpu/instrumentation/gtpin-instrumentation.c

```diff
@@ -284,20 +277,8 @@ findOrAddKernelModule
   strcat(path, ".");
   strncat(path, kernel_name_hash, strlen(kernel_name_hash));
 
-  uint32_t module_id = 0;
-
-  hpcrun_loadmap_lock();
-  load_module_t *module = hpcrun_loadmap_findByName(path);
-  if (module == NULL) {
-    module_id = hpcrun_loadModule_add(path);
-    load_module_t *lm = hpcrun_loadmap_findById(module_id);
-    hpcrun_loadModule_flags_set(lm, LOADMAP_ENTRY_ANALYZE);
-  } else {
-    // Find module
-    module_id = module->id;
-  }
-  hpcrun_loadmap_unlock();
-  uint64_t kernel_name_id = get_numeric_hash_id_for_string(kernel_name, (size_t)kernel_name_len);
+  bool mark_used = true;
```

Comment:
This constant does not need to be a variable. Using `true /* mark_used */` in the argument list is equivalently readable. (At the very least make it `const`).

Reply:
done